features/GEM051_Presence-of-geometric-context.feature

```diff
@@ -1,26 +1,38 @@
 @industry-practice
 @GEM
-@version1
+@version2
 @E00020
 Feature: GEM051 - Presence of Geometric Context
 The rule verifies that a geometric context is present in the model.
 
 
-  Scenario: Agreement on having at least one geometric representation context - IFC2X3
+    Scenario Outline: Agreement on having at least one geometric representation context
 
-    Given A model with Schema "IFC2X3"
-    Given An IfcProject
-    Given Its attribute RepresentationContexts
+      Given A model with Schema "<Schema>"
+      Given An <Entity>
+      Given Its attribute RepresentationContexts
 
-    Then Assert existence 
-    Then Its entity type is 'IfcGeometricRepresentationContext' including subtypes
+      Then Assert existence
+      Then Its entity type is 'IfcGeometricRepresentationContext' including subtypes
 
+      Examples:
+        | Schema  | Entity     |
+        | IFC2X3  | IfcProject |
+        | IFC4    | IfcContext |
+        | IFC4X3  | IfcContext |
```

Comment:
```suggestion
        | IFC4.3  | IfcContext |
```

Reply:
I was a bit surprised by this, but see now that this is handled in https://github.com/buildingSMART/ifc-gherkin-rules/blob/f7dcaac762beb0c613d46f29fe94ca8aa778f9dd/features/steps/givens/attributes.py#L120

Just a heads up that IFC 4.4 is going to be available in the foreseeable future. We might want to do this a bit more generically. For inspiration see https://github.com/IfcOpenShell/IfcOpenShell/blob/6cb063ebd2e4f79b1c679b30e31af82d679b5cf5/src/ifcopenshell-python/ifcopenshell/file.py#L409-L420 We can create our own function that conditionally uses `"."` or `"X"` based on the major version.

But this is also a reminder that of course any inconsistency we introduce on ourselves is going to be have to be maintained indefinitely.

The good news is that everything applicable to IFC4.3 is almost for sure also applicable to IFC4.4 so we can just globally replace `IFC4.3` with `IFC4.3 or IFC4.4`.